test/gui/shared/scripts/pageObjects/Activity.py

```diff
@@ -90,3 +90,55 @@ def isResourceBlackListed(self, context, filename):
             return True
         except:
             return False
+
+    def isResourceExcluded(self, context, filename):
+        try:
+            fileRow = squish.waitForObject(
+                {
+                    "column": 1,
+                    "container": names.oCC_IssuesWidget_tableView_QTableView,
+                    "text": filename,
+                    "type": "QModelIndex",
+                },
+                context.userData['lowestSyncTimeout'] * 1000,
+            )["row"]
+            squish.waitForObjectExists(
+                {
+                    "column": 6,
+                    "row": fileRow,
+                    "container": names.oCC_IssuesWidget_tableView_QTableView,
+                    "text": "Excluded",
+                    "type": "QModelIndex",
+                },
+                context.userData['lowestSyncTimeout'] * 1000,
+            )
+
+            return True
+        except:
+            return False
+
+    def checkResourceNotExist(self, context, filename):
+        squish.waitForObject(names.settings_OCC_SettingsDialog)
+
+        result = squish.waitFor(
+            lambda: self.isResourceNotVisible(context, filename),
```

Comment:
can we do this here? I gues we can
```suggestion
            lambda: not self.isResourcePresent(context, filename),
```

Reply:
this doesn't work.